torch/csrc/autograd/function.cpp

```diff
@@ -35,96 +35,33 @@ AnomalyMetadata* Function::metadata() noexcept {
   }
   return anomaly_metadata_.get();
 }
-
-/*
- * Fix for #5534: prevent stack overflow on deletion of deep computation graph
```

Comment:
This seemed like a perfectly good comment that we should still keep ;)

Reply:
Reasonable